tests/acl/test_acl.py

```diff
@@ -268,7 +296,7 @@ def create_or_remove_acl_table(duthost, acl_table_config, setup, op):
     for sonic_host_or_asic_inst in duthost.get_sonic_host_and_frontend_asic_instance():
         namespace = sonic_host_or_asic_inst.namespace if hasattr(sonic_host_or_asic_inst, 'namespace') else ''
         if op == "add":
-            logger.info("Creating ACL table: \"{}\" in namesspace {}".format(acl_table_config["table_name"], namespace))
+            logger.info("Creating ACL table: \"{}\" in namespace {} on device {}".format(acl_table_config["table_name"], namespace, duthost))
```

Comment:
Should this be `duthost.hostname`?

Reply:
I made a recent change to mult_asic_sonic_host so this should display the hostname: https://github.com/Azure/sonic-mgmt/blob/1b53581d1d32239a5b133b8c9000c0911746e698/tests/common/devices/multi_asic.py#L48